src/Component/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
```

Comment:
- [ ]  Please your app is not rendering, kindly fix the issue and resubmit for review. See screenshot below for clarifications.

![rendering](https://user-images.githubusercontent.com/65852621/154018965-782ab14b-608d-4c5d-add1-44d1c71e3d02.png)